### DISCUSSION TITLE
Chromebook UI Issues

### DISCUSSION THREAD

**Discussion by lafreniereluc** [supermerill/SuperSlicer#1857] [General]
Hi, I'm finally giving SuperSlicer a go after trying Cura/Prusa Slicer. I'm running this on a Chromebook (Google Pixel Slate, i5) via Linux.

Everything seems to work perfectly minus a few UI bugs. I'm wondering if this is something that should be fixed in the software or if it's some setting I can change (but I suspect the former).

1. Whenever I try to select files, trying to open an STL for example, the file system navigation doesn't respond to the mouse at all. It does however respond to keyboard navigation just fine. It's a little odd, but the mouse won't allow you to click on anything.
2. Many of the fonts are too small for some reason. For example, when going through the various options in the left hand column for Extruder, etc. That column has very small text as compared to the rest of the UI/menus.

Merci!

**Comment by supermerill** [comment]
Did you use the GTK2 or GTK3 binaries?

**Reply by lafreniereluc** [reply]
No, sorry. I'm a noob when it comes to this stuff. I couldn't figure out what "GTK" means. I looked and couldn't find an explanation anywhere. I'm guessing this is some sort of Linux terminology that I'm just not familiar with. I can definitely give that a try tonight.

**Reply by supermerill** [reply]
On the release page, there is two Linux appimage, one for GTK2, and the other one is for GTK3.
Download both.
Does the issue is here for both?

**Reply by lafreniereluc** [reply]
Sorry for the delay. Family life happened... lol

So I tried the Linux 64 tar file instead of the Ubuntu version. Since I'm on ChromeOS, it apparently runs a version of Debian. That seems to have fixed access to menus, which is great. 

But many options are still tiny. Here's a screenshot, observe the options on the left: https://photos.app.goo.gl/mnyfQsgrLXgsoRNB9

**Reply by supermerill** [reply]
mmm. I can see how, but i wonder why.

**Reply by lafreniereluc** [reply]
I had a look through scaling settings, fonts, etc. Nothing affected that. I do have my display scaled to 80%.. but I put it back to 100% to see... no effect. It just made everything bigger indeed, but the "ratio" between the small font and the bigger font/buttons remained as such.

At least it looks functional. I'll try and use it for the next few days.

**Reply by supermerill** [reply]
these text aren't done by the ui lib. prusa created a custom class to print them manually. But they should follow the scaling.

**Reply by lafreniereluc** [reply]
OK, the gtk2 one looks good and normal. I just tried it for fun. I'm going to use/test that one.
